assets/app/data/crowbar/services/crowbar-backup.factory.js

```diff
@@ -1,19 +1,24 @@
 (function() {
 
     angular
-        .module('crowbarData.upgrade')
-        .factory('upgradeBackupFactory', upgradeBackupFactory);
+        .module('suseData.crowbar')
+        .factory('crowbarBackupFactory', crowbarBackupFactory);
 
-    upgradeBackupFactory.$inject = ['$q', '$http', '$filter', 'COMMON_API_V2_HEADERS'];
+    crowbarBackupFactory.$inject = ['$q', '$http', '$filter', 'COMMON_API_V2_HEADERS'];
     /* @ngInject */
-    function upgradeBackupFactory($q, $http, $filter, COMMON_API_V2_HEADERS) {
+    function crowbarBackupFactory($q, $http, $filter, COMMON_API_V2_HEADERS) {
         var factory = {
             create: createBackup,
```

Comment:
In the end it would be used like: `crowbarBackupFactory().createBackup()`. Don't you think it's redundant, considering we're already operating over a backup factory?